packages/polywrap-uri-resolvers/polywrap_uri_resolvers/static_resolver.py

```diff
@@ -0,0 +1,50 @@
+from typing import List, cast
+from polywrap_core import IUriResolutionStep, Wrapper, IWrapPackage, UriResolutionResult, IUriResolver, UriPackageOrWrapper, Uri, Client, IUriResolutionContext, UriPackage, UriWrapper
+from polywrap_result import Result
+
+from .helpers import UriResolverLike
+
+
+class StaticResolver(IUriResolver):
+    uri_map: dict[str, UriPackageOrWrapper]
+
+    def __init__(self, uri_map: dict[str, UriPackageOrWrapper]):
+        self.uri_map = uri_map
+
+    @staticmethod
+    def _from(static_resolver_likes: List[UriResolverLike]) -> "StaticResolver":
```

Comment:
This should be class method instead. Let's name this something else. (Ex: construct_from or from_list, from_array?)

Reply:
I think it make sense to keep it as a static method, but the name indeed need to be changed; will rename it to `from_list`